ecommerce/serializers.py

```diff
@@ -1,11 +1,46 @@
 """ ecommerce serializers """
+import pytz
 from django.templatetags.static import static
 from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+from rest_framework.fields import IntegerField
+from rest_framework.validators import UniqueValidator
 
 from courses.models import Course, CourseRun
 from courses.serializers import CourseRunSerializer
 from ecommerce import models
 from ecommerce.api import latest_product_version, latest_coupon_version
+from ecommerce.models import CouponPaymentVersion, CouponPayment, Coupon, Product
+
+
+class DateTimeTzField(serializers.DateTimeField):
```

Comment:
Is there a reason to do this instead of just the regular `DateTimeField` when the field values should be stored in utc?

Reply:
For some reason I was initially getting warnings about naive date-times in the console log last week, but not anymore, so changed to regular `DateTimeField`s.